src/main/java/org/zalando/nakadi/domain/SubscriptionCursor.java

```diff
@@ -40,8 +43,8 @@ public boolean equals(final Object o) {
     @Override
     public int hashCode() {
         int result = super.hashCode();
-        result = 31 * result + eventType.hashCode();
-        result = 31 * result + cursorToken.hashCode();
+        result = 31 * result + (eventType != null ? eventType.hashCode() : 0);
```

Comment:
I check it for null here only because the validation for some reason calls hashCode before it check properties for null.